ide/coqide/wg_ScriptView.ml

```diff
@@ -361,6 +361,10 @@ object (self)
     } in
     Gobject.set prop obj show
 
+  method select_all () =
+    if self#is_focus then
+      self#buffer#select_range self#buffer#start_iter self#buffer#end_iter;
+
```

Comment:
Did you look at the debugger panel, too?  In master I can copy the call stack but not variable values.  But that's more likely something I should fix.

Reply:
Indeed I hadn’t looked at the debugger panel. It is now fixed for the call stack. Do you think I should also enable Ctrl-A for variable values?

Reply:
If it's easy, sure, but note that the panel is a tree rather than a being a simple script panel.